src/styles/summernote/font.scss

```
@@ -73,7 +67,8 @@ $sni-font-path: "./font" !default;
 
   @if "\\#{'x'}" == "\\x" {
     @return str-slice("\x", 1, 1) + $character-code;
-  } @else {
+  }
```

Stylelint (17.14.0)

[error] Unexpected newline after "}" of `@if` statement (scss/at-if-closing-brace-newline-after)

(scss/at-if-closing-brace-newline-after)